TripKitAndroid/src/main/java/com/skedgo/android/tripkit/TripPreferences.java

```diff
@@ -13,5 +15,6 @@
    * This option should be used when {@link RegionInfo#transitWheelchairAccessibility()} is true.
    */
   boolean isWheelchairPreferred();
+  Observable<Boolean> hasWheelchairInformation();
```

Comment:
I see you attempted to create an `Observable` to emit latest value of `isWheelchairPreferred` and any of its future changes. However, the actual case we need is only about when `isWheelchairPreferred` has actually changed. So what do you think if we create an `Observable` that only emits when `isWheelchairPreferred` has actually changed? So the usage of `hasWheelchairInformation()` wouldn't require to `skip(1)` anymore which make it a little bit simpler.

Once we follow that way, I suggest to rename `hasWheelchairInformation` into `whenWheelchairPreferenceChanges()` which describes better what's gonna happen if we subscribe the `Observable` it returns. Wdyt?

Speaking of how to produce such `Observable`, one approach I can suggest is like:

```kotlin
prefs.onChanged().filter { it == wheelchairPrefKey }.map { prefs.getBoolean(wheelchairPrefKey, false) }
```

Reply:
Sure. It's really ambiguous when leaving skip(1) to ignore the first wheelchair with no explain.